src/main/java/net/minestom/server/raw_data/RawMaterialData.java

```diff
@@ -0,0 +1,24 @@
+package net.minestom.server.raw_data;
+
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.network.packet.server.play.EntityEquipmentPacket;
+import net.minestom.server.sound.SoundEvent;
+import org.jetbrains.annotations.Nullable;
+
+public class RawMaterialData {
+    public boolean damageable;
```

Comment:
As these are non-final, would it allow someone to edit these values and would that cause any issues if they did?

Reply:
The idea is that this data is loaded from the json file which is non final. So plugins can change the hardness of a block for example by entering this class. Now the class itself should probably be final (not extendable) but the fields are supposted to be accessible. Maybe it is better to use getters and setters. 

Would it cause problems:
- Really just depends on what your changing and how you are changing it.